release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -377,6 +377,34 @@ and no other Reports or ReportQueries depend on the expiring Report, the Meterin
 removes the Report from your cluster at the end of its retention period. For more information,
 see metering Reports xref:../metering/reports/metering-about-reports.adoc#metering-expiration_metering-about-reports[expiration].
 
+[id="ocp-4-6-monitoring"]
+=== Monitoring
+
+[id="ocp-4-6-monitoring-for-user-defined-projects"]
+==== Monitoring for user-defined projects
+
+In {product-title} 4.6, you can enable monitoring for user-defined projects in addition to the default platform monitoring. You can now monitor your own projects in {product-title} without the need for an additional monitoring solution. Using this new feature centralizes monitoring for core platform components and user-defined projects.
+
+With this new feature, you can perform the following tasks:
+
+* Enable and configure monitoring for user-defined projects
+* Create recording and alerting rules that use metrics from your own pods and services
+* Access metrics and alert information through a single, multi-tenant interface
```

Comment:
Access metrics and alerts information

Supose alert here should be alerts

Reply:
Thank you for your suggestion. I have subsequently changed that bullet point as follows:

* Access metrics and information about alerts through a single, multi-tenant interface